test/cases/OverloadedLegacyFactoryFunctionInterface.webidl

```diff
@@ -0,0 +1,4 @@
+[Exposed=Window,
+ LegacyFactoryFunction=OverloadedLegacyFactoryFunction(optional DOMString src),
+ LegacyFactoryFunction=OverloadedLegacyFactoryFunction(long arg1, long arg2)]
```

Comment:
Let's disallow any instance of the attribute appearing more than once, per https://github.com/heycam/webidl/issues/878#issuecomment-803685451.